packages/payment-integration-api/src/payment/payment-method.ts

```diff
@@ -1,7 +1,7 @@
 import PaymentMethodConfig from './payment-method-config';
 import InitializationStrategy from './payment-method-initialization-strategy';
 
-export default interface PaymentMethod {
+export default interface PaymentMethod<T = any> {
```

Comment:
Can we not use `any`? And come up with better generic typing? Or is that too big a task?

Reply:
I just copy and paste the same implementation as we have in core package. I don't think that we should searching about better type in this pr, because the pr has another goal